styles/settings-view.less

```diff
@@ -7,6 +7,7 @@
 
 .settings-view {
   display: flex;
+  overflow: auto;
```

Comment:
@simurai If so, the left navigation will overflow out of whole pane, when pane is narrow,